ESP32LapTimer/src/Comms.cpp

```diff
@@ -77,6 +77,38 @@
 #define RESPONSE_END_SEQUENCE        'x'
 #define RESPONSE_IS_CONFIGURED       'y'
 
+// Extended commands for chorus32
+#define EXTENDED_PREFIX   'E'
+
+#define EXTENDED_ALL_SETTINGS 'a'
+
+#define EXTENDED_RACE_NUM 'R'
+#define EXTENDED_CALIB_MIN 'c'
+#define EXTENDED_CALIB_MAX 'C'
+#define EXTENDED_CALIB_START 's'
+#define EXTENDED_CALIB_STATUS 'S'
+#define EXTENDED_VOLTAGE_TYPE 'v'
+#define EXTENDED_VOLTAGE_CALIB 'V' // uint16
+#define EXTENDED_NUM_MODULES 'M' // half byte
+#define EXTENDED_CALIBRATE_START 'r'
+#define EXTENDED_EEPROM_RESET 'E'
+#define EXTENDED_DISPLAY_TIMEOUT 'D'
+#define EXTENDED_WIFI_CHANNEL 'W' // half byte
+#define EXTENDED_WIFI_PROTOCOL 'w' // half byte
+#define EXTENDED_FILTER_CUTOFF 'F'
+#define EXTENDED_MULTIPLEX_OFF 'm'
+#define EXTENDED_UPDATE_PROGRESS 'U'
```

Comment:
Remove this since it's currently unused. Could always be added back at a later point if a need for it arises.